docs/advanced-data-layouts/signalling.md

```diff
@@ -9,31 +9,44 @@ eleventyNavigation:
 Advanced Data Layout Signalling
 ===============================
 
-:::todo
-- This page needs to be substantially rewritten to get to the point better.
-:::
+[Advanced Data Layouts](..) are a feature of IPLD which gives us "lenses" to view data in another way.
+
+This page is about answering the question: "How do I know when to use an ADL to read data?"
 
 ### How do I know when to use an ADL to read data?
 
-We call this "the signalling problem".
+Trick question.  You don't.
 
-In short: you don't.
+Using an ADL to read data is **_always_ optional**.
+If you don't want to use an ADL to read data, it's still legible --
+just as the [plain data model](/docs/data-model/) content,
+as a [codec](/glossary#codec) already gave it to you.
 
-Since the data composing the "raw", interior data of an ADL is just regular IPLD Data Model
-(it must be, after all, since it's produced by some [Codec](/docs/codecs/), which by definition produces data structures describable by the Data Model),
-then it follows that there's absolutely no way for this data to unambiguously indicate that it needs an ADL in order to be understood.
-If there was, it would imply that there's some kind of "reserved words" in the Data Model,
-which would violate some of our other central goals in IPLD, because it would mean some perfectly normal maps and lists would be invalid IPLD or gain magical meaning that they shouldn't;
-we don't want any of that.
+You can also use different ADLs to read the same data.
+It's like putting on a different pair of glasses -- maybe some do
+a better job of making the data legible than others!
 
-So!  Signalling must come from somewhere else.
+### That wasn't helpful!
```

Comment:
Are these the right first 2 paragraphs of the document for someone looking for 'tell me how to do this?'

would it not be faster to answer their question by leading with the signalling methods below?